### DISCUSSION TITLE
Cannot Export Unpublished Changes To A Branch

### DISCUSSION THREAD

**Discussion by 0AniketMishra** [community/community#50684] [Codespaces] [answered]
### Select Topic Area

Show & Tell

### Body

I am facing some issues regarding one of my codespace , I have exhausted my 120 hours of free usage and now I am not able to export my unpublished changes to a branch. Whenever I try to export it using the "Export Changes to a branch section" it starts and eventually fails to do so. 
[
![Isuue](https://user-images.githubusercontent.com/111168015/226855159-c776d9f7-201b-4bd9-9248-2a407bca514a.png)
](url)

**Comment by MotionCommunications** [comment]
I faced the same issue a while ago, But I did not got any possible solution and I ended up waiting for the next month to enter my codespace again.

**Reply by 0AniketMishra** [reply]
That would waste a lot of time and I don't have that much time to get my codebase back again. I just need my source code as it is precious to me and I think I would never use Codespaces again in my life.

**Reply by rahulkaushik71144** [reply]
I am facing the same issue, I swear I will never ever use codespace in my life again, cause I have to do this work as soon as possible, I can't wait an entire month.

**Comment by osortega** [comment, marked as answer]
Hi @ConnectionsLLC 👋 !
I'm sorry to hear that, I'm working with my team to get you unblocked so you can export your codebase.

I also surfaced this feedback so we can fix the friction in the future.

**Reply by nikant** [reply]
@osortega This is still a thing on 2024? Just keeps happening to me too..  
:/

**Reply by xhjds** [reply]
I am facing this problem too even though I paid for GitHub pro could you unblock me too

**Reply by ayan479** [reply]
Im currently having the same issue.

**Reply by geniusgeek** [reply]
I am currently experiencing thesame problem and need urgent help fixing it

**Reply by iJoseZX** [reply]
i am having the same problem

**Reply by HyperCodec** [reply]
Same problem here but mine isn't even expired

**Reply by hienngul** [reply]
**I have the same problem. I couldn't use the function even before the time was up, and now that the time has passed, the problem still persists.**

**Reply by drok** [reply]
This response was selected as "answer" by @0AniketMishra, but does not answer the problem.

The devs might think the problem is taken care of, but people are still wasting their time diagnosing this unresolved issue.

Please remove the "answer" flag, and escalate this long-term lingering issue to engineering.

I also recommend a 👎🏻 reaction on the non-answer.

**Comment by 8Gitbrix** [comment]
Hi @ConnectionsLLC Your entitlements have been reset so you should be able to access your codespaces again! Let me know if you can access it now.

**Reply by 0AniketMishra** [reply]
I am very thankful to all of your ❤

**Comment by Gaita3** [comment]
Hi @8Gitbrix . Is there a solution now? I am facing the same problem. I cannot export my changes to a new branch.

**Reply by narutophuc113** [reply]
me too

**Reply by Alyzbane** [reply]
same

**Comment by ceffiro** [comment]
@osortega @8Gitbrix I face the same problem. I exhausted my hours and the creation of a new branch with the unpushed changes keeps failing. Can you please help me with that?

**Comment by AndrewMTA** [comment]
Same problem here

**Reply by 8Gitbrix** [reply]
Apologies. I'm no longer in the codespaces org. Please submit a new post raising this issue

**Comment by AmraniCh** [comment]
same problem here

**Comment by outtachrome** [comment]
same problem

**Comment by BorisOskolkov** [comment]
same

**Comment by AmongUsafk** [comment]
same help

**Comment by geniusgeek** [comment]
I am having thesame problem, please i need help to resolve this asap as i am in a hackathon

**Comment by bathini79** [comment]
![Screenshot from 2024-11-26 10-20-37](https://github.com/user-attachments/assets/45c64e4a-6d31-41de-aac3-2e6fd25871df)
Same for me

**Comment by duckling69** [comment]
![image](https://github.com/user-attachments/assets/186abaf2-b6df-4ce6-b0bd-1afa58c53310)
same issue is there a solution?

**Reply by valerialevitskaya1204** [reply]
same(

**Comment by DAYAVKURSE** [comment]
Actual problem. Please help

**Comment by drok** [comment]
Same problem, and I have only used about 1 hour of free codespaces before I decided to fork (codespace-super-fiesta-g4q5q5jq6v2p64q)

When tracing the request with Chrome DevTools, the github.com server eventually responds to `https://github.com/codespaces/super-fiesta-g4q5q5jq6v2p64q/exported` with `408 Request Timeout` 6m35s after the initial `POST` request, following a series of `GET` polls with what looks like exponential-backoff.

This question is marked as "anwered", but the "answer" by @osortega does not in fact answer the problem. Clearing entitlements does not resolve this timeout. This issue should be "unanswered", maybe then it might be "surfaced" to the engineering team.

**Comment by osortega** [comment]
Hello all,
I'm no longer part of the Codespaces team but I would suggest to file an issue through GitHub Support to get this solved

**Comment by andrezin454** [comment]
what a useful discussion

**Comment by Dxuian** [comment]
wow this isnt fixed yet

**Comment by ussrseth** [comment]
still here

**Comment by Saharsh1223** [comment]
still exists... this is what we call a useful conversation

**Comment by aegonmy** [comment]
same issue here

**Comment by burningtnt** [comment]
This issue still exists.

**Comment by drok** [comment]
I have re-submitted this issue at https://github.com/orgs/community/discussions/179801 as it still exists.

**Comment by Serge-sudo** [comment]
same
